src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt

```diff
@@ -0,0 +1,320 @@
+
+                println("alignAssembly: asmFileFull: ${assemblyEntry.asmFasta}, outputFile: $asmSamFile")
+
+                val builder = ProcessBuilder(
+                    "conda",
+                    "run",
+                    "-n",
+                    "phgv2-conda",
+                    "minimap2",
+                    "-x",
+                    "splice",
+                    "-t",
+                    threads.toString(),
+                    "-k",
+                    "12",
+                    "-a",
+                    "-p",
+                    "0.4",
+                    "-N20",
+                    assemblyEntry.asmFasta,
+                    cdsFasta,
+                    "-o",
+                    asmSamFile
+                )
+
+                val redirectError = "${assemblyEntry.outputDir}/minimap2_${justName}_error.log"
+                val redirectOutput = "${assemblyEntry.outputDir}/minimap2_${justName}_output.log"
+                println("redirectError: $redirectError")
+                builder.redirectOutput(File(redirectOutput))
+                builder.redirectError(File(redirectError))
+                println(
+                    " begin minimap assembly Command: " + builder.command().stream().collect(Collectors.joining(" "))
+                );
+                val process = builder.start()
+                val error = process.waitFor()
+                if (error != 0) {
+                    println("minimap2 for assembly ${assemblyEntry.asmFasta} run via ProcessBuilder returned error code $error")
+                    throw IllegalStateException("alignAssembly: error running minimap2 for ${justName}: $error")
+                }
+                // We have the SAM File, call proali to align with anchorwave
+                runAnchorwaveProali(
+                    gffFile,
+                    assemblyEntry.refFasta,
+                    assemblyEntry.asmFasta,
+                    cdsFasta,
+                    assemblyEntry.refSamOutFile,
+                    asmSamFile
+                )
+            }
+        }
+
+    private fun runAnchorwaveProali(
+        gffFile: String,
+        refFasta: String,
+        asmFasta: String,
+        cdsFasta: String,
+        refSam: String,
+        asmSam: String
+    ) {
+
+        val justNameAsm = File(asmFasta).nameWithoutExtension
+        val justNameRef = File(refFasta).nameWithoutExtension
+
+        val anchorsproFile = "${outputDir}/${justNameAsm}_${justNameRef}.anchorspro"
+        // Using just the assembly name.  This facilitates programmatically creating
+        // a GVCF keyfile from the maf keyfiles.  It will be understood that the maf
+        // file name is <assemblyFastaNoExtension>.maf
+        val outputFile = "${outputDir}/${justNameAsm}.maf"
+        val builder = ProcessBuilder(
+            "conda",
+            "run",
+            "-n",
+            "phgv2-conda",
+            "anchorwave",
+            "proali",
+            "-i",
```

Comment:
I know this is what the IntelliJ formatter does, but it might read a bit easier if you keep the variable on the same line as the parameter name.


"-i", gffFile,
"-r", refFasta,

Reply:
I use the IntelliJ automatic formatting all the time.  I'd have to keep moving it back?